src/components/ToMemoriesButton.js

```diff
@@ -20,11 +20,14 @@ const ButtonStyle = styled(Button)`
   display: flex;
   justify-content: space-evenly;
   align-items: center;
-  position: fixed;
-  left: 1.4em;
-  bottom: 1.4em;
   width: 90vw;
   height: 5vh;
+
+  span {
+    letter-spacing: 4px;
```

Comment:
In this case you might want to use rem or em. So if someone change the font size, the letter spacing is still the same.